src/Jellyfin.LiveTv/Listings/XmlTvListingsProvider.cs

```diff
@@ -167,7 +167,7 @@ private static ProgramInfo GetProgramInfo(XmlTvProgram program, ListingsProvider
                 Overview = program.Description,
                 ProductionYear = program.CopyrightDate?.Year,
                 SeasonNumber = program.Episode.Series,
-                IsSeries = program.Episode.Series is not null,
+                IsSeries = program.Episode is not null,
```

Comment:
This is not the correct way to fix this and will not work as what you intend it to. The type definition of `program.Episode` is not nullable and thus this expression is always true. You are just setting `IsSeries == true` for everything.

Reply:
I think I was looking at this [PR](https://github.com/jellyfin/Jellyfin.XmlTv/pull/12) and though this would be a suitable workaround. It was my misunderstanding, but it does what I want it to do, just not for the right reason. Thanks. 

> In Jellyfin [XmlTvListingProvider.cs](https://github.com/jellyfin/jellyfin/blob/5085ae416436a19591ca4e81bf526a06212639ee/Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs#L209) checks `Episode` for null to determine if the program is part of a series or not:
> 
> ```cs
> SeasonNumber = program.Episode?.Series,
> IsSeries = program.Episode != null,
> ```
> 
> The old code initialized `XmlTvProgram.Episode` to a new `XmlTvEpisode` in the constructor, so it could never be null. This results in Jellyfin treating all programs (movies and shows) as shows ("Record series" button is active, program is recorded to the shows recording directory if set, etc).
> 
> Making `Episode` nullable and leaving it null until an episode tag is encountered fixes the behavior in Jellyfin, and besides tests in this repo there was nowhere that uses Episode that doesn't already test it for null.